README.md

```diff
@@ -76,8 +75,11 @@ Some permissive software licenses request but do not require an acknowledgement
 
 * The [zlib] compression library (Jean-loup Gailly, Mark Adler and team)
 
-[slack-badge]: https://devtools-html-slack.herokuapp.com/badge.svg
-[slack]: https://devtools-html-slack.herokuapp.com/
+[matrix]: https://chat.mozilla.org/#/room/#profiler:mozilla.org
+<!-- The following badge works because our chatmo instance is federated. For
+some reason (probably authentication) this doesn't work when using
+chat.mozilla.org -->
+[matrix-badge]: https://img.shields.io/matrix/profiler:mozilla.org?server_fqdn=matrix.org&label=matrix
```

Comment:
Please tell me how you feel about using an external website for this. They say it's "no tracking" but I don't know how we can trust that... After all because this is in the README this will be displayed for every visitor...

Reply:
> <img width="262" alt="image" src="https://user-images.githubusercontent.com/1588648/79878684-52d07800-83b3-11ea-82ca-633f7909ca42.png">

This doesn't seem to work?

Reply:
Dang, this was working yesterday.
digging a bit more, I found https://github.com/badges/shields/issues/4878, it seems to be an issue on shields.io side, or more specifically, an issue between github's image anonymizing service and shields.io's hosting provider; because loading the shield image directly works perfectly for me at the moment, but loading it from github's proxy which is automatically inserted doesn't. (see https://help.github.com/en/github/authenticating-to-github/about-anonymized-image-urls for more information about anonymized image urls)

I'll remove the badge in the mean time, but I subscribed to that issue so that I'll add it back later :-)

Reply:
I see similar problems in other README using badges from other sources too (eg: https://github.com/mozilla-mobile/fenix). Let's see how this evolves...